test/mpi/coll/allred.c

```diff
@@ -18,7 +18,7 @@
 #endif
 
 int count, size, rank;
-int cerrcnt;
+int errs;
```

Comment:
Can separate out the name change to a different commit?  It's unrelated to this.